master/internal/rm/dispatcherrm/dispatcher_resource_manager.go

```diff
@@ -670,7 +672,9 @@ func (*DispatcherResourceManager) GetSlots(*apiv1.GetSlotsRequest) (*apiv1.GetSl
 
 // DefaultNamespace is unsupported.
 func (*DispatcherResourceManager) DefaultNamespace(string) (*string, error) {
-	return nil, rmerrors.ErrNotSupported
+	return nil, status.Error(codes.NotFound,
+		"resource manager type AgentRM does not have a default namespace: "+
```

Comment:
typo: this is dispatcherRM

Reply:
can you just substitute the message for all these funcs with rmerrors.ErrNotSupported? I don't think we need this extra info

Reply:
done